### PR TITLE
Add dummy modal for Sign In in the simulator itself

### LINKED ISSUE #1250: Signup and LogIn should happen in the simulator itself.
**Is your feature request related to a problem? Please describe.**
Consider me a new user , I opened the website and just started playing without noticing the "Sign in" button . Suddenly when I will want to save the project , I will click the button and it will take me out of the page , believe me it's frustating and not okay.

**Describe the solution you'd like**
I want that there should be a cool "modal" , bootsrap's modal is pretty fine . In the modal there should be all sort of login methods , and when the user is signed in , the modal should just go away , updating the simulator without refreshing the page at all.

**Are you working on this? (Yes/No)**
Yes , I have deignned the look of the modal . As understanding the login flow of CircuitVerse will take time , I can't finish it completely as PR , But I will surely show the mentors how I want it to look in proposal with a picture . I will create a basic modal with dummy inputs and a create a PR for sure.

### PR DESCRIPTION
**Pull request by RohitKumarGit**
Fixes #1250

#### Describe the changes you have made in this PR -
I was frustated when the user clicked on sign in button in the simulator , it takes him to external webpage , so I created a modal with login form

**what else I plan to do**
If I get selected then I will understand the login and signup flow , create a signup modal similarly and then implement all the functionality releated to account in the simulator itself , making the User experience awesome

### Screenshots of the changes (If any) -
modalLogin

This appears when the user clicks on the navbar's sign in button :)

### CODE REVIEW

File: app/views/simulator/edit.html.erb [hunk 1]
```diff
@@ -4,20 +4,57 @@
...
+      <div class="form-group form-check">
+        <input type="checkbox" class="form-check-input" id="exampleCheck1">
+        <label class="form-check-label" for="exampleCheck1">Keep me logged in</label>
+
+      </div>
+      <p><em>Don't have an account? <b>SignUp</b></em></p>
+      <button type="submit" class="btn btn-primary">LOG IN</button>
+    </form>
+      </div>
+
```
[the remark this answers is not in the document]
Comment (RohitKumarGit) [reply app/views/simulator/edit.html.erb]: I will fix it for sure

File: app/views/simulator/edit.html.erb [hunk 2]
```diff
@@ -4,20 +4,57 @@
...
+        <label for="exampleInputPassword1">Password</label>
+        <input type="password" class="form-control" id="exampleInputPassword1">
+      </div>
+      <div class="form-group form-check">
+        <input type="checkbox" class="form-check-input" id="exampleCheck1">
+        <label class="form-check-label" for="exampleCheck1">Keep me logged in</label>
+
+      </div>
+      <p><em>Don't have an account? <b>SignUp</b></em></p>
+      <button type="submit" class="btn btn-primary">LOG IN</button>
```
Comment (KaustubhDamania) [inline app/views/simulator/edit.html.erb]: It should be "Sign Up" and not "SignUp"
SImilarly, "Login" instead of "LOG IN"

**Comment by KaustubhDamania** [comment]
Also, share screenshot of the modal in mobile view.

**Comment by KaustubhDamania** [comment]
Screenshot_20200320-202755~01


In mobile view, the title of the simulation, i.e. "Untitled" is partially visible.

File: app/views/simulator/edit.html.erb [hunk 3]
```diff
@@ -4,20 +4,57 @@
...
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+      <form>
+      <div class="form-group">
+        <label for="exampleInputEmail1">Email address</label>
+        <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+        <small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
```
[the remark this answers is not in the document]
Comment (RohitKumarGit) [reply app/views/simulator/edit.html.erb]: okay , i'll fix it.

**Comment by RohitKumarGit** [comment]
> Screenshot_20200320-202755~01
> 
> In mobile view, the title of the simulation, i.e. "Untitled" is partially visible.

The simulator will be disabled in mobile

**Comment by satu0king** [comment]
You should be keeping the aesthetic changes in #1258 and the sign in modal in this PR. Please close this PR and raise a new PR with just the sign in modal.

**Comment by nitin10s** [comment]
We'll be designing a new modal for Login.

**Pull request closed without merging**